dual_net.py

```diff
@@ -387,3 +469,12 @@ def after_run(self, run_context, run_values):
             self.summary_writer.add_summary(
                 weight_update_summaries, global_step)
             self.before_weights = None
+
+parser = argparse.ArgumentParser()
+argh.add_commands(parser, [train])
```

Comment:
can you use the single-form here?  i think it's argh.dispatch_command

Reply:
I want to leave as multi-dispatch so that I can move all the main.py dual_net-specific stuff back into dual_net.py. Do you think that's a good idea?

Reply:
which stuff is that?
my mental model is:
main.py: 
basic operations on models.  setup for training (bootstrap, train, validate) or setup to run inference (gtp/selfplay/eval), push models around (freeze, update-global-step).  All the entrypoints you'd want to use in scripts, basically.
rl_loop.py
main.py ops with the boilerplate filled in around cloud-based paths, and codifying the idea of a sequential series of models.

so running 'dual_net.py' seems like a new category of stuff -- how are you thinking about this category?  Is there a different taxonomy that makes more sense?